src/Microsoft.ML/Models/BinaryClassificationMetrics.cs

```diff
@@ -162,6 +164,12 @@ internal static List<BinaryClassificationMetrics> FromMetrics(IHostEnvironment e
         /// </summary>
         public ConfusionMatrix ConfusionMatrix { get; private set; }
 
+        /// <summary>
+        /// For cross validation gives fold number or standard deviation or average across all metrics.
```

Comment:
>fold number or standard deviation or average across all metrics [](start = 39, length = 63)

let's elaborate, because right now it feels unclear.
Maybe call it 'row tag', and then add a comment that
```
For cross-validation, this is equal to "Fold N" for per-fold metric rows, "Overall" for the average metrics and "STD" for standard deviation.
For non-CV scenarios, this is equal to null.
```
(whatever the actual strings are).